adoc/admin-disaster-recovery-backup.adoc

```diff
@@ -0,0 +1,161 @@
+= Backup
+This chapter describes the backup of {productname}.
+
+== Data To Backup
+* The `skuba`: command-line binary for the running cluster, use to replace nodes from cluster.
+* The skuba cluster folder: directory where used for running `skuba` commands. Contains cluster configurations and authentication files.
+* The kubernetes folder of every node in cluster: directory contains kubernetes configurations.
++
+```
+/etc/kubernetes
+```
+* The kubelet folder of every node in cluster: directory contains kubelet configurations.
++
+```
+/var/lib/kubelet
+```
```

Comment:
Sorry, I did not see the detailed steps to back up these. Could you please elaborate more? :bowing_man:

Reply:
Will add some info to this.